.github/workflows/deployment-e2e.yml

```diff
@@ -1,16 +1,16 @@
-name: Deployment E2E
+name: test
```

Comment:
Is it possible to keep the existing names which look more descriptive?

Reply:
it is descriptive, you see it as `test / e2e / cf-worker` now